lib/aptible/cli/subcommands/logs.rb

```diff
@@ -25,6 +28,149 @@ def logs
               ENV['ACCESS_TOKEN'] = fetch_token
               exit_with_ssh_portal(op, '-o', 'SendEnv=ACCESS_TOKEN', '-T')
             end
+
+            desc 'logs_from_archive --bucket NAME | --aws-region REGION | ' \
+                 '--stack NAAME --decryption-keys ONE [OR MORE] | ' \
+                 '--download-location | --string-matches ONE [OR MORE] |' \
+                 '--start-date YYYY-MM-DD | --end-date YYYY-MM-DD',
+                 'Retrieves container logs from an S3 archive in your own ' \
+                 'AWS account. You must provide your AWS credentials via ' \
+                 'the environment variables AWS_ACCESS_KEY_ID and ' \
+                 'AWS_SECRET_ACCESS_KEY'
+
+            # Required to retrieve files
+            option :region,
+                   desc: 'The AWS region your S3 bucket resides in',
+                   type: :string, required: true
+            option :bucket,
+                   desc: 'The name of your S3 bucket',
+                   type: :string, required: true
+            option :stack,
+                   desc: 'The name of the Stack to download logs from',
+                   type: :string, required: true
+            option :decryption_keys,
+                   desc: 'The Aptible-provided keys for decryption. ' \
+                         '(Comma separated if multiple)',
+                   type: :array, required: true
+
+            # For identifying files to download
+            option :string_matches,
+                   desc: 'The strings to match in log file names.',
+                   type: :array
+            option :app_id,
+                   desc: 'The Application ID to download logs for.',
+                   type: :numeric
+            option :database_id,
+                   desc: 'The Database ID to download logs for.',
+                   type: :numeric
+            option :proxy_id,
+                   desc: 'The Endpoint ID to download logs for.',
```

Comment:
intended? looks like a mismatch unless i'm misreading something (`Endpoint` == `proxy`?). I looked else where in the codebase for the word `proxy_id` and could not find it. Is there any reason we can't use endpoint_id as that seems to be the name we're using for the actual resource?  asking (either because i see this mismatch and/or to learn why)

Reply:
Good catch. This is a lot like the `Account` model being an "Environment".    They're "Endpoints", but sometimes in code referred to as `proxy`.  The options should be `--endpoint-id` and I'll fix that.